utils/progressbar/progressbarmng.go

```diff
@@ -166,14 +166,20 @@ func (bm *ProgressBarMng) DoneTask(prog *TasksWithHeadlineProg) {
 	prog.tasksProgressBar.bar.IncrBy(int(diff))
 }
 
-func (bm *ProgressBarMng) NewTasksProgressBar(totalTasks int64, colour Color, taskType string) *TasksProgressBar {
+func (bm *ProgressBarMng) NewTasksProgressBar(totalTasks int64, colour Color, windows bool, taskType string) *TasksProgressBar {
```

Comment:
The following comment applies to all functions that create progres bars - 
It seems unnecessary to send the `windows` argument to these functions, when they can determine whether the OS is Windows or not by themselves.